_includes/banner.html

```diff
@@ -1,4 +1,3 @@
-<span id="github-ribbon"><a href="https://github.com/swcarpentry/bc">Fork me on GitHub</a></span>
```

Comment:
Was this removed intentionally? Should that be a different pull request if you think the ribbon shouldn't be there?